selfdrive/car/honda/carcontroller.py

```diff
@@ -127,7 +128,18 @@ def __init__(self, dbc_name, CP, VM):
   def update(self, CC, CS):
     actuators = CC.actuators
     hud_control = CC.hudControl
-    hud_v_cruise = hud_control.setSpeed * CV.MS_TO_KPH if hud_control.speedVisible else 255
+
+    hud_v_cruise = hud_control.setSpeed * CV.MS_TO_MPH if hud_control.speedVisible else 255
+    if self.CP.carFingerprint in HONDA_BOSCH_RADARLESS:
+      conversion = CV.MS_TO_KPH if CS.is_metric else CV.MS_TO_MPH
+      hud_v_cruise = hud_control.setSpeed * conversion if hud_control.speedVisible else 255
+      if CC.longActive and actuators.longControlState == LongCtrlState.stopping:
+        self.stopping_counter += 1
+        if self.stopping_counter > 400:
+          hud_v_cruise = 252
```

Comment:
can you make a global variable with a comment for this magic number of what it does?

Reply:
I'm assuming the magic number in reference is 400 and not 252, or should we make both variables?

Reply:
400 is very clearly 4 seconds, I meant what that set speed did

Reply:
Does showing stopped change any behavior like turn off the engine?

Reply:
Silly me, 400 wasn't 4 seconds to me because it would start counting down before it came to a stop. 252 is the stopped message which we now don't need.